Content.Shared/Nutrition/Components/HungerComponent.cs

```diff
@@ -63,6 +63,15 @@ public sealed partial class HungerComponent : Component
         { HungerThreshold.Dead, 0.0f }
     };
 
+    /// <summary>
+    /// The minimum and maximum amount that hunger can be randomly set at when an entity is spawned
+    /// </summary>
+    [DataField("hungerMin"), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField]
```
do this for all datafields